staging/app/network.tf

```diff
@@ -2,7 +2,8 @@ data "aws_availability_zones" "available" {
 }
 
 resource "aws_vpc" "staging" {
-  cidr_block = "172.17.0.0/16"
+  cidr_block           = "172.17.0.0/16"
```

Comment:
are these values ok to keep in the repo?

Reply:
Yeah, this is the private network inside of the VPC so the block of addresses we use doesn't really matter (as long as it's one of the [private networks](https://en.wikipedia.org/wiki/Private_network#Private_IPv4_addresses)).